docs/installation.rst

```diff
@@ -53,7 +50,7 @@ Supported Python Versions
 -------------------------
 
 Python 3.6 and 3.7 are officially supported, including testing during
-development and packaging. Support for Python 2.7 has been dropped as of 
+development and packaging. Support for Python 2.7 has been dropped as of
 August 6, 2019. Other Python versions, such as 3.8 and 3.5 and older, may
```

Comment:
Since we are testing for 3.8, we may change this to `Other Python versions, such as 3.9 and 3.5 and older,`

Reply:
Thanks for pointing this out. I have added your suggestion.